### PR TITLE
Range splitting: remove hardcoded chunk limit

### PR DESCRIPTION
**Pull request by matyax**
This PR removes the hardcoded chunk limit we originally added to prevent indefinitely blocking the browser. The initial implementation was in orders of magnitude slower than the current one, so there doesn't seem to be any reason to have one.

**Which issue(s) does this PR fix?**:

Part of https://github.com/grafana/grafana/issues/61768

**Special notes for your reviewer**:

Users should be able to make larger requests than before.

### CODE REVIEW

File: public/app/plugins/datasource/loki/querySplitting.ts [hunk 1]
```diff
@@ -179,23 +174,28 @@ export function runPartitionedQueries(datasource: LokiDatasource, request: DataQ
...
-      partition: partitionTimeRange(false, request.range, request.intervalMs, metricQueries[0].resolution ?? 1),
-    });
-  }
-  if (instantQueries.length) {
-    requests.push({
-      request: { ...request, targets: instantQueries },
-      partition: [request.range],
-    });
+  try {
+    if (logQueries.length) {
```
Comment (gabor) [inline public/app/plugins/datasource/loki/querySplitting.ts:178]: hi, this change may be unintentional?

**Review by gabor — CHANGES_REQUESTED** [review]
diff is good (except one maybe-unintentional change), works fine 👍

File: public/app/plugins/datasource/loki/querySplitting.ts (hunk 1 above)
Comment (matyax) [reply public/app/plugins/datasource/loki/querySplitting.ts:178]: Leftover, removed.

**Review by gabor — APPROVED** [review]
LGTM 👍

**Pull request merged**